### DISCUSSION TITLE
Save Data Transfer?

### DISCUSSION THREAD

**Discussion by jonahkel** [leoetlino/project-restoration#216] [Q&A] [answered]
If I already have save data for vanilla Majora's Mask 3d, will it transfer over to the patch? I'm using a New 3ds console. I'm new to the modding scene so I don't know how game patching works exactly. I have some issues with the game and your patch seems like the perfect solution, but I don't want to install it if I have to start all over.

Thanks for helping! Also, thank you for taking the time and effort to make this mod and give people a "best of both worlds" Majora's Mask experience!

**Comment by jonahkel** [comment, marked as answer]
I got impatient and tried it out and it indeed uses my same save data!

**Reply by henrydarklordofdeath** [reply]
Ah thanks so much for this
